### PR TITLE
style update and titleupdate

### PR DESCRIPTION
**Pull request by JulienDelemarle**

### CODE REVIEW

**Review by Maenhout — DISMISSED** [review]
RAS

File: src/components/Request.jsx [hunk 1]
```diff
@@ -132,9 +144,10 @@ export default function Request() {
               placeholder="Tell us what a SuperHero you would like to be!"
             />
           </label>
+          {submitting && <Loading>Submitting Form...</Loading>}
           <Abutton>
             <Sbutton type="submit" className="button">
-              <p>SUBMIT</p>
+              <p2>SUBMIT</p2>
```
Comment (LoicBrassart) [inline src/components/Request.jsx]: This tag doesn't exist ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Almost no Evil

File: src/components/Request.jsx [hunk 2]
```diff
@@ -102,28 +107,27 @@ export default function Request() {
...
     <>
       <Article>
-        <title>Request your own SuperHero</title>
-        {submitting && <Loading>Submitting Form...</Loading>}
+        <title>Request your own SuperHero !</title>
+
         <form className="RequestForm" onSubmit={handleSubmit}>
           <label htmlFor="username">
-            <p>LastName</p>
+            <h1>LastName</h1>
```
Comment (Maenhout) [inline src/components/Request.jsx]: No more than one h1, per page. Use of h2, h3 etc. request

**Review by Maenhout — COMMENTED** [review]
Titration to review

**Review by Maenhout — APPROVED** [review]
RAS

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**